fix-moment-imports.sh

```
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# 批量替换moment导入为dayjs导入的脚本
+# 用于修复jshERP前端项目中的moment依赖问题
+
+echo "开始批量替换moment导入为dayjs导入..."
+
+# 进入前端项目目录
+cd /Users/macmini/Desktop/jshERP-0612-Cursor/jshERP-web
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)